README.md

```diff
@@ -1258,6 +1258,7 @@ trivy:
     DOCKER_DRIVER: overlay2
     # See https://github.com/docker-library/docker/pull/166
     DOCKER_TLS_CERTDIR: ""
+    IMAGE: trivy-ci-test:$CI_COMMIT_SHA
```

Comment:
This can be the same as in https://github.com/aquasecurity/trivy/pull/376 ?

Reply:
> Maybe it makes sense to simply embed/link #376 in the yaml to avoid repeating ourselves here.

I don't think so. This description on README is the easiest way for everyone but the CI template under `/contrib` (#376) is for GitLab Container Scanning and its report feature integration, never for every newcomers to GitLab CI and Trivy.

> This can be the same as in #376 ?

The above is why #375 and #376 should not be the same.